umh.docs.umh.app/content/en/docs/getstarted/managingthesystem.md

```diff
@@ -1,142 +1,247 @@
+title: 2. Managing the System
+menuTitle: 2. Managing the System
+description: Learn how to manage your UMH instance with the Management Console.
+weight: 2000
+---
 
-  In this chapter, we will guide you through the installation of the Management 
-  Console and your first local instance of the UMH. We will also explain
-  how the basic components work together, by creating a first Node-RED flow and 
-  a Grafana dashboard. Check out the image below for an 
-  overview:
+In this chapter, you will learn how to monitor, manage and configure your UMH
+instance with the Management Console.
 
-![Untitled](/images/getstarted/managingTheSystem/getStartedUMHSimplifiedpng.png)
+At this stage, you should have already installed the UMH on your device. If you
+have not done so, please follow the steps in the [Installation](/docs/getstarted/installation)
+chapter first.
 
-  The United Manufacturing Hub is a collection of microservices, which work
-  together, to provide a complete solution for your manufacturing needs. From 
-  data collection to contextualization and visualization, or data storage. 
-  For rare cases that require a special feature, you can add it to your UMH
-  stack by creating your own custom microservice.
+## A Few Words About the Communication
 
-## 1. Install the UMH
+Now that you have connected a UMH instance to the Management Console, you might
+be curious about how the Management Console communicates with the instance.
 
+The Management Companion is a microservice that runs in conjunction with the UMH.
+It plays a crucial role in linking the instance to the Management Console. This
+includes gathering the status of the device and the various components within
+the system.
 
-1. Download and install [Docker Desktop](https://www.docker.com/products/docker-desktop)
-   for your operating system. This is only 
-   necessary for local installations.
-   
+The diagram below illustrates the communication flow between the Management
+Console and the instance:
 
-2. Follow the tutorial in the Management Console to create your first local 
-   instance of the United Manufacturing Hub. Once the installation is finished,
-   press the blue **Finish** button the right bottom corner of the Console. 
+![Communication between the Management Console and the instance](/images/getstarted/managingTheSystem/simpleInfrastructureDiagram.png?width=80%)
 
+## Overview of Your Instances
 
-{{% notice info %}}
-A local installation functions similar to an installation on an external device,
-like an edge device or a VM. The benefit is, that you can easily set it up
-without any additional hardware and learn how to use it with simulated data.
+On the left side of the Management Console, you can view the list of your
+instances. If you have just installed the UMH, you should see only one instance
+in the list.
+
+The status of your instance is indicated by color: green means everything is
+working properly, while yellow indicates that there may be an issue.
```

Comment:
Orange indicates that the instance is experiencing connection issues. If an instance is experiencing connection issues for over 10 seocnds, it will be considered offline.

Reply:
todo: double check